clusters/app.ci/prow-monitoring/retester_servicemonitor.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
```

Comment:
If you use `PodMonitor` instead, you would not need to define the service below.
Example:

https://github.com/openshift/release/blob/93026636adf9eb636e8e5364a02e876003d72eea/clusters/app.ci/assets/dptp-controller-manager.yaml#L58